src/guidelines/theme/shadows.stories.tsx

```diff
@@ -0,0 +1,78 @@
+import type { Meta, StoryObj } from "@storybook/react";
+import React from "react";
+import styled from "styled-components";
+import { getContrastYIQ, spacing } from "./index";
+import { theme } from "./index";
+import { heading1 } from "./typographies";
+
+const meta: Meta = {};
+
+export default meta;
+
+type Story = StoryObj;
+
+const Title = styled.h1`
+  ${heading1};
```

Comment:
Pour une unité entre les titres des différentes story
```suggestion
  ${heading1};
  margin: ${sizing(24)} 0 ${sizing(16)};
```